spec/features/model_test_train_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
```

Comment:
This should actually be a _model_ test, not a _feature_ test -- you want to test the behavior of the model, not a feature of your app. Take a look here for how to test your associations: https://learn.launchacademy.com/teams/boston-24/curricula/on-campus-boston-24/lesson_groups/week_5:_databases/lessons/sinatra-testing-activerecord-validations-associations